src/service/user/getWrittenProjects.ts

```
@@ -0,0 +1,61 @@
+import { prisma } from '../../config/prisma';
+
+const VALID_PROJECT_STATES: Exclude<ProjectState, null>[] = [
+  'ALL', 'PENDING', 'APPROVAL', 'REJECTED', 'MODIFY', 'WRITING'
+];
+
+const isValidProjectState = (state: string): state is ProjectState => {
+  return VALID_PROJECT_STATES.includes(state as ProjectState);
+};
+
+export const getWrittenProjects = async (
+  req: AuthenticatedRequest<{}, {}, {}, { state?: ProjectState }>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)